### PR TITLE
Added support for prismatic joint in URDF importer

### PR DESCRIPTION
**Pull request by michalpelka**
Signed-off-by: Michał Pełka <michalpelka@gmail.com>

### CODE REVIEW

**Comment by michalpelka** [comment]
It addresses #192

File: Code/Source/RobotImporter/URDF/JointsMaker.cpp [hunk 1]
```diff
@@ -98,6 +99,26 @@ namespace ROS2
                 jointComponent = followColliderEntity->CreateComponent<PhysX::EditorFixedJointComponent>();
             }
             break;
+        case urdf::Joint::PRISMATIC:
+            { // Implemented as Hinge with limit set to -2*PI and 2*PI deg. Works fine in the Engine
```
Comment (adamdbrw) [inline Code/Source/RobotImporter/URDF/JointsMaker.cpp]: I am not sure if I understand this comment. How is it relevant here?

**Comment by adamdbrw** [comment]
One comment, otherwise looks good!

File: Code/Source/RobotImporter/URDF/JointsMaker.cpp (hunk 1 above)
Comment (adamdbrw) [reply Code/Source/RobotImporter/URDF/JointsMaker.cpp]: Perhaps it was referring to Continuous?

**Review by adamdbrw — APPROVED** [review]

**Pull request merged**